released/packages/coq-coqprime/coq-coqprime.1.0.5/opam

```diff
@@ -0,0 +1,24 @@
+opam-version: "2.0"
+maintainer: "thery@sophia.inria.fr"
+homepage: "http://coqprime.gforge.inria.fr/"
+bug-reports: "http://coqprime.gforge.inria.fr/"
+dev-repo: "git+https://github.com/thery/coqprime.git"
+license: "LGPL"
```

Comment:
```suggestion
license: "LGPL-2.1-only"
```
Can the license be a bit more precise? Based on the repo, this is what I could deduce.